alembic/versions/945688b6e665_new_migration.py

```diff
@@ -0,0 +1,50 @@
+"""New migration
+
+Revision ID: 945688b6e665
+Revises: 59fd579e174c
+Create Date: 2024-10-31 14:20:46.450528
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '945688b6e665'
+down_revision: Union[str, None] = '59fd579e174c'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('author',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=True),
+    sa.Column('bio', sa.String(length=1023), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_author_id'), 'author', ['id'], unique=False)
+    op.create_table('book',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('title', sa.String(length=255), nullable=False),
+    sa.Column('summary', sa.String(length=255), nullable=False),
```

Comment:
The `summary` column in the `book` table is set to `nullable=False`. Ensure this aligns with your data requirements, as it will require every book entry to have a summary.